packages/core/src/components/StarRating.tsx

```diff
@@ -62,7 +91,7 @@ export const SEED_DATA = {
   name: "Star Rating",
   tag: "StarRating",
   description: "A star rating component",
-  category: COMPONENT_TYPES.basic,
+  category: COMPONENT_TYPES.button,
   props: {
     starSize: createStaticNumberProp({
```

Comment:
To make this work with Draftbit's state system, you'll also want to remove the "rating" prop below and add (I think):

```suggestion
    fieldName: createFieldNameProp({
      defaultValue: "ratingValue", // this is the name of the variable declared on the screen in Draftbit
      handlerPropName: "onPress", // the change handler prop in this component
      valuePropName: "rating", // the value prop in this component
    }),
    starSize: createStaticNumberProp({
```

Reply:
updated